### ISSUE TITLE
On the previous version I could scan websites in a certain order. Ex. The order they were in my sidebar 

### ISSUE THREAD

**Issue by hapomykal** [sneakypete81/updatescanner#174]
Thanks for creating a new issue!

Before continuing, please use the search box in the header above to check if someone has already reported it.

To help explain the issue, please include the following:
 * Firefox version:
 * Update Scanner version:

#### Steps to reproduce the problem:
[Include webpage URL, if applicable]

#### What actually happened:
[Include a screenshot]

**Comment by jfg001** [comment]
Hello
"right click on the folder to sort the items in alphabetical order"
Do you think this feature can be implemented in the near future?
I went back to firefox 56 and updatescanner 3 to be able to classify the items.
thank you for your answer

**Comment by hapomykal** [comment]
Thank you for your diligence.  
John 14:6 “I am the way and the truth and the life."

Sent from Yahoo Mail on Android 
 
  On Fri, Dec 29, 2017 at 8:21 AM, jfg001<notifications@github.com> wrote:   
Hello
"right click on the folder to sort the items in alphabetical order"
Do you think this feature can be implemented in the near future?
I went back to firefox 56 and updatescanner 3 to be able to classify the items.
thank you for your answer

**Comment by sneakypete81** [comment]
@jfg001 see #143.

@hapomykal - I agree that it would be useful to sort the sidebar alphabetically. But I'm not sure why changing the order of scanning is useful.

**Comment by hapomykal** [comment]
Thank you.  I found out how to scan in the order I desire... 1. drag links into the scan  position I desire (in the side bar - that alone doesn't alter scan order) 2. Backup links/scans 3. Then restore.  Upon restore, links are scanned in order (descending from top to bottom)
Just a note: I like to scan in a certain order because some scans are of more important to me and I  want to know (and pursue) those results first. 
Thank you for update scanner and all the effort spent . I'm not a computer guy... so the tool helps me greatly.
HaroldJesus is the way, truth, and LIFE.
Sent from Yahoo Mail on Android 
 
  On Sun, Mar 11, 2018 at 3:33 AM, sneakypete81<notifications@github.com> wrote:   
Closed #174.

**Comment by sneakypete81** [comment]
In the upcoming 4.3.0 release scanning will now be done in the order of the scan list.

**Comment by hapomykal** [comment]
Hello. I'm not sure this email will still be active, because I have not contacted you in about 5 years. I have returned to use update scanner but I noticed that I'm not able to scan websites that require a password, like Facebook and other similar. Is there a work around? Thank you again for update scanner and what it provides.

Sent from Yahoo Mail on Android 
 
  On Sun, Mar 11, 2018 at 8:43 AM, ***@***.******@***.***> wrote:   Thank you.  I found out how to scan in the order I desire... 1. drag links into the scan  position I desire (in the side bar - that alone doesn't alter scan order) 2. Backup links/scans 3. Then restore.  Upon restore, links are scanned in order (descending from top to bottom)
Just a note: I like to scan in a certain order because some scans are of more important to me and I  want to know (and pursue) those results first. 
Thank you for update scanner and all the effort spent . I'm not a computer guy... so the tool helps me greatly.
HaroldJesus is the way, truth, and LIFE.
Sent from Yahoo Mail on Android 
 
  On Sun, Mar 11, 2018 at 3:33 AM, ***@***.***> wrote:   
Closed #174.
